spec/lib/rmt/scc_spec.rb

```diff
@@ -131,4 +141,66 @@ def scc
       expect { other_repo_without_token.reload }.not_to raise_error
     end
   end
+
+  describe '#export' do
+    let(:path) { '/tmp/usb' }
+
+    context 'without SCC credentials' do
+      before do
+        allow(Settings).to receive(:scc).and_return OpenStruct.new
+      end
+
+      it 'raises an error' do
+        expect { described_class.new.export(path) }.to raise_error RMT::SCC::CredentialsError, 'SCC credentials not set.'
+      end
+    end
+
+    context 'with SCC credentials' do
+      before do
+        allow(Settings).to receive(:scc).and_return(OpenStruct.new(username: 'me', password: 'groot'))
+      end
+
+      it 'makes sure path exists' do
+        expect(File.directory?(path)).to be true
+      end
+
+      %w[orders products repositories subscriptions].each do |data|
+        it "writes #{data} file to path" do
+          FakeFS.with_fresh do
+            described_class.new.export(path)
+            expect(File.exist?(File.join(path, "organizations_#{data}.json")))
+          end
+        end
+      end
+    end
+  end
+
+  describe '#import' do
+    let(:path) { '/tmp/usb' }
+
+    context 'with bad path or missing files' do
+      it 'raises an error before it touches the database' do
```

Comment:
I've had `/tmp/usb` directory on my machine for some reason, and this test was failing.
It would be better if it was either wrapped into FakeFS block or used a random directory name.